.github/workflows/angular-deploy-github-pages.yml

```
@@ -14,26 +14,8 @@ jobs:
-        run: npm run build -- --output-path ../dist
-        working-directory: frontend/
-
-      - name: Run angular/karma tests
-        run: npm run test-ci
-        working-directory: frontend/
-
-      - name: Run vitest
-        run: npm run test-vitest
-        working-directory: frontend/
+      - name: Test
+        uses: .github/actions/test.yml
```

actionlint (1.7.12)

[error] specifying action ".github/actions/test.yml" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)